core/src/test/java/io/questdb/griffin/O3FailureTest.java

```diff
@@ -825,7 +821,7 @@ public void testPartitionedDataAppendOODataNotNullStrTailIndexAllocateFailConten
     @Test
     public void testPartitionedDataAppendOODataNotNullStrTailParallel() throws Exception {
         counter.set(174 + 45);
-        executeWithPool(2, O3FailureTest::testPartitionedDataAppendOODataNotNullStrTailFailRetry0, ffAllocateFailure);
+        executeWithPool(2, O3FailureTest::testPartitionedDataAppendOOPrependOODatThenRegularAppend0, ffAllocateFailure);
```

Comment:
Could you clarify why we need this change? Looks like we're changing the scenario under test here, don't we?

Reply:
the test was incorrect, I try this in the console and it works just fine:

```sql

drop table if exists x;
drop table if exists append;

create table x as (
    select
        cast(x as int) i,
        rnd_symbol('msft','ibm', 'googl') sym,
        round(rnd_double(0)*100, 3) amt,
        to_timestamp('2018-01', 'yyyy-MM') + x * 720000000 timestamp,
        rnd_boolean() b,
        rnd_str('ABC', 'CDE', null, 'XYZ') c,
        rnd_double(2) d,
        rnd_float(2) e,
        rnd_short(10,1024) f,
        rnd_date(to_date('2015', 'yyyy'), to_date('2016', 'yyyy'), 2) g,
        rnd_symbol(4,4,4,2) ik,
        rnd_long() j,
        timestamp_sequence(500000000000L,100000000L) ts,
        rnd_byte(2,50) l,
        cast(null as binary) m,
        rnd_str(5,16,2) n,
        rnd_char() t
        from long_sequence(510)
    ), index(sym) timestamp (ts) partition by DAY;

create table append as (
    select
        cast(x as int) i,
        rnd_symbol('msft','ibm', 'googl') sym,
        round(rnd_double(0)*100, 3) amt,
        to_timestamp('2018-01', 'yyyy-MM') + x * 720000000 timestamp,
        rnd_boolean() b,
        rnd_str('ABC', 'CDE', null, 'XYZ') c,
        rnd_double(2) d,
        rnd_float(2) e,
        rnd_short(10,1024) f,
        rnd_date(to_date('2015', 'yyyy'), to_date('2016', 'yyyy'), 2) g,
        rnd_symbol(4,4,4,2) ik,
        rnd_long() j,
        timestamp_sequence(518300000010L,100000L) ts,
        rnd_byte(2,50) l,
        rnd_bin(10, 20, 2) m,
        rnd_str(5,16,2) n,
        rnd_char() t
        from long_sequence(100)
) timestamp (ts) partition by DAY;
insert into x select * from append;
```
so I took the liberty of correcting it.